### PR TITLE
DCA-185: Refactor the logic of creating reuslts directory

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-185

### CODE REVIEW

File: app/selenium_ui/conftest.py [hunk 1]
```diff
@@ -28,19 +28,19 @@
...
     if 'TAURUS_ARTIFACTS_DIR' in os.environ:
-        return os.environ.get('TAURUS_ARTIFACTS_DIR')
+        return Path(os.environ.get('TAURUS_ARTIFACTS_DIR'))
     else:
         # TODO we have error here if 'results' dir does not exist
         results_dir_name = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
-        pytest_run_results = f'results/{results_dir_name}_local'
-        os.mkdir(pytest_run_results)
+        pytest_run_results = Path(f'results/{results_dir_name}_local')
+        os.makedirs(pytest_run_results, exist_ok=True)
```
Comment (astashys) [inline app/selenium_ui/conftest.py:36]: what do you think about `Path("/my/directory").mkdir(parents=True, exist_ok=True)`?

File: app/selenium_ui/conftest.py [hunk 2]
```diff
@@ -28,19 +28,19 @@
...
         return pytest_run_results  # in case you just run pytest
 
 
 # create selenium output files
 current_results_dir = __get_current_results_dir()
-selenium_results_file = Path(current_results_dir + '/selenium.jtl')
-selenium_error_file = Path(current_results_dir + '/selenium.err')
+selenium_results_file = Path(str(current_results_dir) + '/selenium.jtl')
```
Comment (astashys) [inline app/selenium_ui/conftest.py:42]: please use operator `/`

File: app/selenium_ui/conftest.py [hunk 3]
```diff
@@ -28,19 +28,19 @@
 
 def __get_current_results_dir():
     if 'TAURUS_ARTIFACTS_DIR' in os.environ:
-        return os.environ.get('TAURUS_ARTIFACTS_DIR')
+        return Path(os.environ.get('TAURUS_ARTIFACTS_DIR'))
     else:
         # TODO we have error here if 'results' dir does not exist
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/conftest.py:33]: This TODO is redundant in the current implementation.

Comment (mmizin) [reply app/selenium_ui/conftest.py]: Done

**Pull request merged**